keptn/lifecycle-toolkit
# test: add KUTTL collectors for scheduling gate tests

# Description

I added a few collectors for now to discuss the issue further. Output is available in 
[test-result.txt](https://github.com/keptn/lifecycle-toolkit/files/13925484/test-result.txt)


Fixes #2700

# How to test
We have to manually fail a test and then check if the collectors are giving the expected output or not. In this case,  `scheduling gate integration tests` are failing in my local machine without modifying anything. So I get above output described in [test-result.txt](https://github.com/keptn/lifecycle-toolkit/files/13925484/test-result.txt)
Possible command:
```bash
make integration-test-scheduling-gates-local
```
